### PR TITLE
DOCS-1257: Add more FUSE guidance

### PR DESCRIPTION
**Pull request by andf-viam**
Add prominent mention of the need to install FUSE v2 in applicable places:
- For Linux installs, via the include file `static/include/install/install-linux.md`, which is included in parent pages:
	- Installation Guide
	- Raspberry Pi, Beaglebones, and SK-TDA4VM board Prepare Guides.
- Expanded troubleshooting section accordingly. Linked above here as well.
- Added need to install FUSE v2 to Requirements section on three modules that I was able to confirm require it:
	- `csi-cam`
	- `rplidar`
	- `cartographer`

Questions

- Is there a good way to tell if FUSE 2 is installed, what version its running, that is platform agnostic? I would love something like `sshfs -v` or similar, if anyone knows of such an approach.
- Are there other modules that ship as AppImages that I missed??

### CODE REVIEW

File: static/include/install/install-linux.md
Comment (andf-viam) [inline static/include/install/install-linux.md:1]: See PR desc for the list of pages where this include file is ... included.

File: static/include/install/install-linux.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
 `viam-server` is distributed for Linux as an [AppImage](https://appimage.org/).
-The AppImage is a single, self-contained binary that runs on 64-bit Linux systems running the `aarch64` or `x86_64` architectures, with no need to install any dependencies.
+The AppImage is a single, self-contained binary that runs on 64-bit Linux systems running the `aarch64` or `x86_64` architectures, with no need to install any dependencies (except for FUSE, which is required by the AppImage format).
 
 To install `viam-server` on a Linux computer:
 
+1. Install FUSE version 2 if it is not already installed on your Linux system.
```
[the remark this answers is not in the document]
Comment (Otterverse) [reply static/include/install/install-linux.md:6]: `find /usr -name libfuse.so.2` should work in most cases I think.

File: docs/appendix/troubleshooting.md [hunk 2]
```diff
@@ -62,11 +62,24 @@ This is only required for the first `ssh` connection you make to a newly-imaged
 
 ### AppImages require FUSE to run
 
-**Description:** `viam-server` is distributed for Linux as an [AppImage](https://appimage.org/), which relies on FUSE (Filesystem-in-Userspace).
+**Related Error:** `dlopen(): error loading libfuse.so.2`
+
+**Description:** `viam-server` is distributed for Linux as an [AppImage](https://appimage.org/), which requires FUSE (Filesystem-in-Userspace) version 2.
 FUSE is included in almost all modern Linux distributions by default, but some older Linux distros or minimal installs might not provide it out of the box.
+For example, the latest Raspberry Pi OS (Debian GNU/Linux 12 bookworm) does not include FUSE with its default installation.
```
Comment (Otterverse) [inline docs/appendix/troubleshooting.md:69]: Is this confirmed? Or is it just FUSE 2 not installed (and it actually does have FUSE 3?)

File: static/include/install/install-linux.md [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+   For example, on Ubuntu (including Raspberry Pi OS), run the following commands:
```
Comment (Otterverse) [inline static/include/install/install-linux.md:7]: Raspberry Pi OS is NOT Ubuntu.

**Review by Otterverse — CHANGES_REQUESTED** [review]
Left a few notes around commands and wording.

File: docs/appendix/troubleshooting.md (hunk 2 above)
Comment (andf-viam) [reply docs/appendix/troubleshooting.md]: Does have FUSE 3, I'll fix this language, thank you!

**Comment by andf-viam** [comment]
Thanks @Otterverse , updated with your feedback! Thanks for your wisdom!

**Review by Otterverse — APPROVED** [review]
Much better, thanks!

**Review by npentrel — APPROVED** [review]

**Pull request merged**